core/creators/themedui.js

```diff
@@ -384,6 +389,61 @@ CKEDITOR.replaceClass = 'ckeditor';
 		return editor;
 	}
 
+	// Delay editor creation if given element is detached from DOM.
+	//
+	// Require `config.delayIfDetached` to be `true`.
+	//
+	// If `config.delayIfDetached_callback` is declared as function, it will be invoked with single argument:
+	// function, that should be called to create editor.
+	//
+	// Otherwise: periodically (with `setInterval`) run check if element is attached to DOM and create editor automatically.
+	//
+	// ```js
+	//	CKEDITOR.replace( detachedEditorElement, {
+	//		delayIfDetached: true,
+	//		delayIfDetached_interval: 50,
+	//		delayIfDetached_callback: registerCallback
+	//		}
+	//	} );
+	// ```
+	//
+	// @param {CKEDITOR.element} element The DOM element.
+	// @param {String} mode CKEditor mode (source/wysiwyg).
+	// @param {Object} [config] The specific configuration to apply to this
+	// editor instance. Configuration set here will override the global CKEditor settings.
+	// @param {String} [data] Initial value for the instance.
+	function delayCreationOnDetachedElement( element, mode, config, data ) {
+		if ( !config || !config.delayIfDetached || !element.isDetached() ) {
+			return false;
+		}
+
+		if ( config.delayIfDetached_callback ) {
+			config.delayIfDetached_callback( function() {
+				createInstance( element, config, data, mode );
+			} );
+
+			return true;
+		}
+
+		var interval = config.delayIfDetached_interval === undefined ? CKEDITOR.config.delayIfDetached_interval : config.delayIfDetached_interval;
+
+		CKEDITOR.warn( 'editor-delayed-creation', {
+			interval: interval
```

Comment:
As mentioned before, I propose to change the data passed:
```suggestion
			mode: 'interval -  ' + interval + ' ms'
```

or inversely (so the result would be `interval - 50 ms` or `50 ms interval` - the first is more coherent with callback scenario, but the second one feels more natural, so I'm leaving the decision up to you).

Reply:
The first scenario indeed sounds more coherent as `mode` property value.